<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Not Malware</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelGif">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>227</width>
      <height>285</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>390</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  H-hi thewe...
Do you th-think I could have youw
cwedit cawd infowmation, p-pwease?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCardNum">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Card number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExpDate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Expiry date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecCode">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Security code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputCardNum">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>95</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>5449083208698666</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputExpDate">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>135</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>08/2025</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSecCode">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>175</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>345</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelThanks">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>225</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Th-thanks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
